client/src/NewVideo.js

```diff
@@ -0,0 +1,73 @@
+import { useState } from "react";
+
+export default function NewVivdeo({ setRefreshVideos }) {
+  const [formData, setFormData] = useState({
```

Comment:
It's good to group these related pieces of data together, nicely done!

(Though I'm not sure about `id` and `rating`? They seem to be unused in this component?)

Reply:
Thanks, I have minimalized passing formData